config/software/make.rb

```diff
@@ -17,8 +17,7 @@
 name "make"
 default_version "4.1"
 
-license "libpng"
-license_file "LICENSE"
+license :project_license
```

Comment:
The only issue I can think off is libpng license not being available on some platform?  Make by itself is a very "stallman"-y project and it going to be GPL but we aren't linking against or using Make's source.